<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>768</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QListWidget" name="tellingListWidget">
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tellingTabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab1">
         <attribute name="title">
          <string>Getelde producten</string>
         </attribute>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QTableWidget" name="tableWidget1">
            <property name="alternatingRowColors">
             <bool>false</bool>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="columnCount">
             <number>5</number>
            </property>
            <column>
             <property name="text">
              <string>Product ID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Product naam</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Originele hoeveelheid</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Getelde hoeveelheid</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Datum geteld</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab2">
         <attribute name="title">
          <string>Gemiste producten</string>
         </attribute>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QTableWidget" name="tableWidget2">
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="columnCount">
             <number>3</number>
            </property>
            <column>
             <property name="text">
              <string>Product ID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Product naam</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Aantal in stock</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="markAsCountedButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Markeer als geteld</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Afdrukken</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBestand">
    <property name="title">
     <string>Bestand</string>
    </property>
    <addaction name="actionNieuwe_telling"/>
    <addaction name="separator"/>
    <addaction name="actionAfsluiten"/>
   </widget>
   <addaction name="menuBestand"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionData_synchroniseren">
   <property name="text">
    <string>Data synchroniseren</string>
   </property>
  </action>
  <action name="actionAfsluiten">
   <property name="text">
    <string>Afsluiten</string>
   </property>
  </action>
  <action name="actionAfdrukken">
   <property name="text">
    <string>Afdrukken</string>
   </property>
  </action>
  <action name="actionNieuwe_telling">
   <property name="text">
    <string>nieuwe telling</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionAfsluiten</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
